playground/backend/internal/db/datastore/datastore_db.go

```diff
@@ -388,16 +400,9 @@ func (d *Datastore) GetExampleOutput(ctx context.Context, id string) (string, er
 }
 
 func (d *Datastore) GetExampleLogs(ctx context.Context, id string) (string, error) {
```

Comment:
GetExampleLogs, GetExampleGraph don't need transaction, it's a single Get request which is atomic